app/src/main/java/org/apache/taverna/mobile/activities/DashboardMainActivity.java

```diff
@@ -89,13 +101,11 @@ protected void onCreate(Bundle savedInstanceState) {
             FragmentManager fragmentManager = getSupportFragmentManager();
```

Comment:
Make a class in utils for Fragment Transaction and use for fragment transaction.
Example : https://github.com/therajanmaurya/Popular-Movies-Stage-1/blob/master/app/src/main/java/opensource/popularmoviesstage1/utils/ActivityUtils.java